app/index.js

```diff
@@ -362,9 +377,15 @@ class GlobalBindings {
         window.matrixWidget.setAlwaysOnScreen(true)
 
         // Register all channels, recursively
+	  
+	if(channelName.indexOf("/") != 0) {
+	  channelName = "/"+channelName;
+	}
         const registerChannel = (channel, channelPath) => {
           this._newChannel(channel)
+ 	  console.log(channelPath)
```

Comment:
This looks like a debug message. If it is, please remove it, otherwise indent with spaces.